src/+replab/CharacterTable.m

```diff
@@ -0,0 +1,112 @@
+classdef CharacterTable < replab.Obj
+% Describes the character table of a group
+
+    properties (SetAccess = protected)
+        group % replab.Group: group represented by character table
```

Comment:
Wrap type with parentheses. And for RepLAB classes, use backticks so that Sphinx crosslinks them; but package names need a `+`.

So here, 
```
%    group (`+replab.Group`): Group ...
```